contentcuration/contentcuration/migrations/0119_auto_20200915_1839.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-09-15 18:39
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+
+import contentcuration.models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+    dependencies = [
+        ('contentcuration', '0118_auto_20200904_1737'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AddIndex(
+                    model_name="channel",
+                    index=models.Index(fields=["name"], name="channel_name_idx"),
+                ),
+            ],
+            database_operations=[
+                # operation to run custom SQL command (check the output of `sqlmigrate`
+                # to see the auto-generated SQL, edit as needed)
+                migrations.RunSQL(
+                    sql='CREATE INDEX CONCURRENTLY "{index_name}" ON "contentcuration_channel" ("name");'.format(
```

Comment:
Ah nice, I was gonna mention that we need to make the indexes concurrently :D looks like you got this covered